test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
Yea, and I'm still of the opinion everything, including social should end up as active :)